### ISSUE TITLE
iOS VoiceOver doesn't read settings toggles as toggles

### ISSUE THREAD

**Issue by LeandraH** [digitalfabrik/integreat-app#2763]
### Describe the Bug

When using iOS VoiceOver to navigate the settings in the app, the system reads the text for the toggle buttons ("Push notifications Receive push notifications for the latest local news") without telling me that there is a toggle button, what state the button is in, and how to interact with it.

Android TalkBack does do exactly that.

![Image](https://github.com/digitalfabrik/integreat-app/assets/18513943/c185a61a-da0b-4aa2-98e1-42399758b47f)

### Steps to Reproduce

1. Use iOS VoiceOver
2. Go to settings
3. Listen to description of the toggle buttons

### Expected Behavior

I would like VoiceOver to tell me the text of the button, that it is a toggle, what state the toggle is in,<s> and how to change that state (VoiceOver usually says to double tap to interact or something along those lines)</s> _Edit: that's only TalkBack, VoiceOver only tells you that it's a button_

### Actual Behavior

VoiceOver tells me the text of the button.

### Additional Information

<details>
  <summary>Traceback</summary>

```

```

</details>
